src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseSign1Message.cs

```diff
@@ -198,6 +308,16 @@ private static async Task<int> CreateCoseSign1MessageAsync(Stream content, byte[
             return writer.Encode(buffer);
         }
 
+        /// <summary>
+        /// Verifies that the signature is valid for the content using the specified key.
+        /// </summary>
+        /// <param name="key">The private key used to sign the content.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must match the value provided during signing.</param>
+        /// <returns><see langword="true"/> if the signature is valid; otherwise, <see langword="false"/>.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="key"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="key"/> is of an unsupported type.</exception>
+        /// <exception cref="InvalidOperationException">Content is detached from this message, use an overload that accepts a detached content.</exception>
+        /// <exception cref="CryptographicException"><see cref="CoseHeaderLabel.Algorithm"/> was missing, was incorrectly formatted, was not one of the supported values or doesn't match with the algorithms supported by the specified <paramref name="key"/>.</exception>
```

Comment:
This is going to look weird in docs.

# Exceptions

## [CryptographicException](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.cryptographicexception?view=dotnet-plat-ext-6.0)
[CoseHeaderLabel.Algorithm](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.cryptographicexception?view=dotnet-plat-ext-6.0) was missing, ...

---


Oh no! Some static member was deleted from the assembly!  Oh, wait.

```suggestion
        /// <exception cref="CryptographicException">
        ///   <para>
        ///     <see cref="ProtectedHeaders" /> does not have a value for the <see cref="CoseHeaderLabel.Algorithm"/> header.
        ///   </para>
        ///   <para>-or-</para>
        ///   <para>
        ///     The algorithm protected header was incorrectly formatted.
        ///   </para>
        ///   <para>-or-</para>
        ///   <para>
        ///     The algorithm protected header was not one of the values supported by this implementation.
        ///   </para>
        ///   <para>-or-</para>
        ///   <para>
        ///     The algorithm protected header doesn't match with the algorithms supported by the specified <paramref name="key"/>.
        ///   </para>
        /// </exception>
```